### PR TITLE
docs(cn): conditional-rendering

### PR DESCRIPTION
**Pull request by Alienover**

### CODE REVIEW

File: src/pages/learn/conditional-rendering.md [hunk 1]
```diff
@@ -202,15 +202,15 @@ return (
...
 ```
 
-You can read it as *"if `isPacked` is true, then (`?`) render `name + ' ✔'`, otherwise (`:`) render `name`."*)
+你可以认为，*“如果 `isPacked=true` 时，则（`?`）渲染 `name + ' ✔'`，否则（`:`）渲染 `name`。”*
 
-<DeepDive title="Are these two examples fully equivalent?">
+<DeepDive title="这两个例子完全相同吗？">
 
-If you're coming from an object-oriented programming background, you might assume that the two examples above are subtly different because one of them may create two different "instances" of `<li>`. But JSX elements aren't "instances" because they don't hold any internal state and aren't real DOM nodes. They're lightweight descriptions, like blueprints. So these two examples, in fact, *are* completely equivalent. [Preserving and Resetting State](/learn/preserving-and-resetting-state) goes into detail about how this works.
+如果你之前是习惯面向对象开发的，你可能会认为上面的两个例子略有不同，因为其中一个可能会创建两个不同的 `<li>` “实例”。但 JSX 元素不是“实例”，因为它们没有内部状态也不是真是的 DOM 节点。它们只是一些简单的描述，就像图纸一样。所以上面这两个例子，事实上，是完全相同的。在 [状态的保持和重置](/learn/preserving-and-resetting-state) 里会深入探讨其原因。
```
Comment (Alienover) [inline src/pages/learn/conditional-rendering.md:209]: Will update the `Preserving and Resetting State` translation after someone translated it.

File: src/pages/learn/conditional-rendering.md [hunk 2]
```diff
@@ -308,46 +308,46 @@ export default function PackingList() {
...
+结合 `if` 语句，当 `isPacked` 为 `true` 时，将 JSX 表达式的值重新赋值给 `itemContent`：
 
 ```js
 if (isPacked) {
   itemContent = name + " ✔";
 }
 ```
 
-[Curly braces open the "window into JavaScript".](/learn/javascript-in-jsx-with-curly-braces#using-curly-braces-a-window-into-the-javascript-world) Embed the variable with curly braces in the returned JSX tree, nesting the previously calculated expression inside of JSX:
+[大括号 - JavaScript 的“新世界”。](/learn/javascript-in-jsx-with-curly-braces#using-curly-braces-a-window-into-the-javascript-world) 将变量用大括号嵌入在返回的 JSX 树中，来嵌套计算好的表达式与 JSX：
```
Comment (Alienover) [inline src/pages/learn/conditional-rendering.md:342]: Will update the `Curly braces open the "window" into JavaScript" after someone translated it.

File: src/pages/learn/conditional-rendering.md (hunk 1 above)
Comment (qinhua) [reply src/pages/learn/conditional-rendering.md:209]: Really awkward.
Feels the need to do a synchronization after the relevant chapter is completed and we'd better synchronize to the issue #611  in time.

Comment (Alienover) [reply src/pages/learn/conditional-rendering.md:209]: Yeah. @QC-L can we standardize the translation for each chapter title and publish them to #609

**Comment by Alienover** [comment]
Close and move to the branch based on `main`

File: src/pages/learn/conditional-rendering.md [hunk 3]
```diff
@@ -1,24 +1,26 @@
...
+* 如何根据不同条件包含或者去掉部分 JSX
+* 一些你会在 React 代码库里遇到的常用的条件语法快捷表达式
 
 </YouWillLearn>
 
-## Conditionally returning JSX
+## 选择性地返回 JSX {#conditionally-returning-jsx}
 
-Let’s say you have a `PackingList` component rendering several `Item`s, which can be marked as packed or not:
+假设有一个 `PackingList` 组件，里面渲染多个 `Item` 组件，每个物品均可被标记为已被打包与否：
```
Comment (watonyweng) [inline src/pages/learn/conditional-rendering.md:23]: 假设有一个 `PackingList` 组件，里面渲染多个 `Item` 组件，每个物品可标记为打包与否：

File: src/pages/learn/conditional-rendering.md [hunk 4]
```diff
@@ -52,9 +54,9 @@ export default function PackingList() {
 
 </Sandpack>
 
-Notice that some of the `Item` components have their `isPacked` prop set to `true` instead of `false`. You want to add a checkmark (✔) to packed items if `isPacked={true}`.
+可以注意到的是，有些 `Item` 组件的 `isPacked` 属性是被设为 `true` 而不是 `false`。你想在那些满足 `isPacked={true}` 条件的物品旁加上一个勾选符号（✔）。
```
Comment (watonyweng) [inline src/pages/learn/conditional-rendering.md:57]: 需要注意的是，有些 `Item` 组件的 `isPacked` 属性是被设为 `true` 而不是 `false`。你可以在那些满足 `isPacked={true}` 条件的物品旁加上一个勾选符号（✔）。

File: src/pages/learn/conditional-rendering.md [hunk 5]
```diff
@@ -52,9 +54,9 @@ export default function PackingList() {
...
 
-You can write this as an [`if`/`else` statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else) like so:
+你可以用 [if/else 语句](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else) 去实现：
```
Comment (watonyweng) [inline src/pages/learn/conditional-rendering.md:59]: 你可以使用 [`if`/`else` 语句](https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Statements/if...else) 去来判断：

File: src/pages/learn/conditional-rendering.md (hunk 4 above)
Comment (Alienover) [reply src/pages/learn/conditional-rendering.md:57]: Updated at https://github.com/reactjs/zh-hans.reactjs.org/pull/636/commits/0041e9d92ace312e849e5b2727abd43dd93f482d#diff-b0f0e8aaf6751d4aaa0a27097e1022b7ead0e43827f66866e1a5ccc8585f1e4fR57

File: src/pages/learn/conditional-rendering.md (hunk 5 above)
Comment (Alienover) [reply src/pages/learn/conditional-rendering.md:59]: Updated at https://github.com/reactjs/zh-hans.reactjs.org/pull/636/commits/0041e9d92ace312e849e5b2727abd43dd93f482d#diff-b0f0e8aaf6751d4aaa0a27097e1022b7ead0e43827f66866e1a5ccc8585f1e4fR59

**Pull request closed without merging**